### DISCUSSION TITLE
I can't access GitHub Education, We require applicants of Technical School of the University of Mendoza to use one of these school-issued email addresses to apply:

### DISCUSSION THREAD

**Discussion by FacundoAguinaga** [community/community#137757] [GitHub Education]
### Hi everyone,

i have a problem, "We require applicants of Technical School of the University of Mendoza to use one of these school-issued email addresses to apply:
etec.um.edu.ar

Please [add and verify your school-issued email address ](https://github.com/settings/emails)in your account settings – or a contact email if you do not have one. Once your email is verified, refresh this page."

my mail is alumno.etec.um.edu.ar and github does not accept it, the message is still there. help.

**Comment by alx-mp4** [comment]
Hi @FacundoAguinaga, it seems that they haven't added that as an option for this university's emails. I would suggest opening a GitHub support ticket.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
